package.json

```diff
@@ -111,6 +111,7 @@
     "@polymer/paper-toolbar": "^3.0.1",
     "@polymer/paper-tooltip": "^3.0.1",
     "@polymer/polymer": "^3.4.1",
+    "@tensorflow/tfjs": "^2.3.0",
     "@vaadin/vaadin-grid": "^5.6.6",
     "d3": "5.7.0",
     "dagre": "^0.8.5",
```

Comment:
Can we drop the `"weblas": "^0.9.1"` on L128, and remove weblas from yarn.lock, too?  Or does something else still use it?

Reply:
> Plan afterwards: will remove dependency on weblas.

As described in the description, I have a plan to remove weblas everywhere afterwards (tf_imports/js.bzl/etc).